lib/archethic/metrics/collector.ex

```diff
@@ -12,17 +12,17 @@ defmodule ArchEthic.Metrics.Collector do
   @doc """
   Get the list of Node IP addresses
   """
-  @spec retrieve_node_ip_addresses() :: list(:inet.ip_address())
-  def retrieve_node_ip_addresses do
-    Enum.map(P2P.authorized_nodes(), & &1.ip)
+  @spec retrieve_node_ip_addresses_and_http_ports() :: list({:inet.ip_address(), port()})
+  def retrieve_node_ip_addresses_and_http_ports do
```

Comment:
What do you think of: `get_node_endpoints` ?

Reply:
LGTM